Add note to clarify column-level encryption timing (#19450)
cockroachdb/docs: src/current/v23.2/column-level-encryption.md

Cockroach Labs measured baseline performance in a 3-node CockroachDB cluster running on three [`n1-standard-4` machines on GCP](https://cloud.google.com/compute/docs/general-purpose-machines#n1_machines).

> Without using `encrypt` or `decrypt`, the following statement generally ran in 60-80 ms (note: in this and the following examples, the timings we report are for 10,000 operations):

Reviewer: Would this be accurate & simpler?

```suggestion
Without using `encrypt` or `decrypt`, the following statement (which represents 10,000 operations) generally ran in 60-80 ms:
```

Author: I don't think it's accurate because the following statement does not execute 10,000 operations (AIUI); instead, it was executed 10,000 times and the timings are based on that number of executions
